### ISSUE TITLE
Windows compilation fails in third party code

### ISSUE THREAD

**Issue by Zylann** [godotengine/godot#14461]
Godot 3.0 master
Windows 10 64 bits, Visual Studio 2015 compiler

Compilation fails:
```
d:\projets\info\godot\engine\godot_fork\thirdparty\thekla_atlas\nvmesh\param\atlas.h(67) : error C4716: 'nv::Atlas::setFailed' : must return a value
```

Because a bool function in atlas.h doesn't return a value:
https://github.com/godotengine/godot/blob/master/thirdparty/thekla_atlas/nvmesh/param/Atlas.h#L67

**Comment by akien-mga** [comment]
> Godot 3.0 master

That's why you should specify this commit :)

Fixed by 690c05e6 already, the line you linked does not need to return a value, it's `void` ;)

**Comment by Zylann** [comment]
Wow that was quick, I was still editing the issue xD
